src/commands/job_cmds/update.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const env = require('../../environment');
+const logger = require('../../logger');
+const Zeus = require('../../sdk');
+
+module.exports = {
+  command: ['update', 'u'],
+  description: 'Add/update a job',
+
+  builder: /* istanbul ignore next */ yargs =>
+    yargs
+      .option('url', {
+        description: 'Custom URL',
+        type: 'string',
+        alias: 'u',
+      })
+      .option('ref', {
+        description: 'Commit hash',
+        type: 'string',
+        alias: 'r',
+      })
+      .option('build-label', {
+        description: 'Custom build label',
+        type: 'string',
+        alias: 'bl',
```

Comment:
_nit_: It is unusual to have two long-option aliases. How about `-B` and `-J`, or entirely removing the alias?